index.html

```diff
@@ -4,7 +4,32 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/style.css" />
     <title>Quiz App</title>
   </head>
-  <body></body>
+  <body>
+    <footer class="footer">
+      <nav class="nav-bar">
+        <ul class="navigation">
+          <li class="home-icon">
```

Comment:
Do you use the `home-icon` anywhere in your project? I recommend not putting code in here, when it is not used. ;-)

Reply:
i plan to use it, if not i will delete it